scm-webapp/src/main/java/sonia/scm/ManagerDaoAdapter.java

```diff
@@ -46,8 +46,6 @@ public void modify(T object, Function<T, PermissionCheck> permissionCheck, Aroun
     if (notModified != null) {
       permissionCheck.apply(notModified).check();
 
-      doThrow().violation("type must not be changed").when(!notModified.getType().equals(object.getType()));
```

Comment:
Actually not since we may edit external users too. If they login again through their external system the user date will be reset though. But it still can be useful.